### DISCUSSION TITLE
Welcome to arcemu Discussions!

### DISCUSSION THREAD

**Discussion by dfighter1985** [arcemu/arcemu#432] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by CyberMist2** [comment]
Is there a discord server for it?

**Reply by dfighter1985** [reply]
Hi there!
Nope, we only have this github forums at the moment.
